src/dapps/store.ts

```diff
@@ -0,0 +1,69 @@
+import { generateValidator, JSONSchema, ValidateFunction } from '../validation'
+
+export type Store = {
+  id: string // urn
```

Comment:
Not necessary for now, but it would be nice to have urns validated in the schema

Reply:
Just daydreaming but for example we could even add utilities like `store.id[3]` to get that section of the urn if necessary or something like a tree to navigate through different hierarchies. dunno 😂

Reply:
yep, tho I dunno how to do that with json schema tbh 😅